runhouse/resources/hardware/cluster_factory.py

```diff
@@ -395,6 +395,12 @@ def ondemand_cluster(
             f"local ~/.rh/config.yaml."
         )
 
+    if autostop_mins == -1:
```

Comment:
don't know if we need this if we change the default to 60 min, bc then this means setting `-1` is intentional?